Description: Remove Tahoma fallback
Origin: vendor, https://bugs.launchpad.net/ubuntu/+bug/434054/comments/8
Bug-Ubuntu: https://bugs.launchpad.net/bugs/434054
Author: Felix Geyer <debfx-pkg@fobos.de>

Index: thaifonts-scalable-0.4.13/fontconfig/89-ttf-thai-tlwg-synthetic.conf
===================================================================
--- thaifonts-scalable-0.4.13.orig/fontconfig/89-ttf-thai-tlwg-synthetic.conf	2009-12-02 00:20:32.000000000 +0700
+++ thaifonts-scalable-0.4.13/fontconfig/89-ttf-thai-tlwg-synthetic.conf	2009-12-02 00:22:22.000000000 +0700
@@ -1,15 +1,6 @@
 <?xml version="1.0"?>
 <!DOCTYPE fontconfig SYSTEM "fonts.dtd">
 <fontconfig>
-	<!-- Fall back missing Tahoma to Waree -->
-	<match target="pattern">
-		<test qual="any" name="family" compare="eq">
-			<string>Tahoma</string>
-		</test>
-		<edit name="family" mode="append" binding="same">
-			<string>Waree</string>
-		</edit>
-	</match>
 	<!-- Fall back missing MS Sans Serif to Loma -->
 	<match target="pattern">
 		<test qual="any" name="family" compare="eq">
